astropy/visualization/stretch.py

```diff
@@ -590,7 +589,7 @@ def inverse(self):
 
 class InvertedHistEqStretch(BaseStretch):
     """
-    Inverse transformation for `~astropy.image.scaling.HistEqStretch`.
```

Comment:
Wut... how did all our checks didn't catch this?

Reply:
It's not public (in `__all__`).

Reply:
But perhaps they should be public.  Not sure if that was intentional or an oversight -- question for @astrofrog.